compiler/rustc_borrowck/src/session_diagnostics.rs

```diff
@@ -148,3 +148,33 @@ pub(crate) enum RequireStaticErr {
         multi_span: MultiSpan,
     },
 }
+
+#[derive(Subdiagnostic)]
+#[label(borrowck_capture_kind_label)]
+pub(crate) struct CaptureVarKind {
+    pub kind_desc: String,
+    #[primary_span]
+    pub kind_span: Span,
+}
+
+#[derive(Subdiagnostic)]
+pub(crate) enum CaptureVarCause {
+    #[label(borrowck_var_borrow_by_use_place)]
+    BrowUsePlace {
```

Comment:
Can you change this from `Brow`? This abbreviation isn't used anywhere else for "borrow".

Reply:
I'm sorry, my eye sliped on it 🥺